Data-Structures/HashTable/HashTable.js

```diff
@@ -0,0 +1,105 @@
+export default class HashTable {
+  constructor(limit = 100) {
+    // Initialize the storage and limit variables
+    this.storage = new Array(limit)
+    this.limit = limit
+  }
+
+  // Hash function:
+  _hash(key, max) {
```

Comment:
Use the [`#` syntax for private properties](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Private_properties). The `max` parameter is also unnecessary; you can just get it as `this.storage.length` instead.